packages/matchbox-site/package.json

```diff
@@ -8,6 +8,7 @@
     "@sparkpost/design-tokens": "^1.0.0-beta.16",
     "@sparkpost/matchbox": "^4.0.0-hibana.15",
     "@sparkpost/matchbox-icons": "^1.1.12",
+    "beautify": "0.0.8",
```

Comment:
this package was last updated 4 years ago :/  and doesn't look like it supports jsx/react

Could we remove this and figure this out in a later ticket? Ideally we would want to run prettier when the code get edited too, like when return is hit or the editor gets blurred, but definitely a small ux enhancement